src/CSET/operators/plot.py

```diff
@@ -261,6 +261,9 @@ def spatial_contour_plot(
         # Use sequence value so multiple sequences can merge.
         sequence_value = cube_slice.coord(sequence_coordinate).points[0]
         plot_filename = f"{filename.rsplit('.', 1)[0]}_{sequence_value}.png"
+        time_coord = cube_slice.coord(sequence_coordinate)
+        times = time_coord.units.num2date(time_coord.points[0])
+        title = f"{("{:4d}{:02d}{:02d} {:02d}:{:02d}".format(times.year, times.month, times.day, times.hour, times.minute))}"
```

Comment:
```suggestion
        time = time_coord.units.num2date(time_coord.points[0])
        title = time.isoformat()
```

The issue causing the tests to fail here is that you have unescaped `"` inside the `"` defining the string boundaries. This is allowed from python 3.12, but not before.

However as we have a datetime object to start with, we can just use the provided method to get a string format of it.